### ISSUE TITLE
about flow accuracy

### ISSUE THREAD

**Issue by ll490187880** [yjxiong/tsn-pytorch#56]
The accuracy of ucf101 (split 1) with flow is only 81%, but I used the official code to extract the flow. And the acc is 87% on the paper.Can you tell me what is wrong? @yjxiong

**Comment by yjxiong** [comment]
I can't. You have to add more details here.

**Comment by imnotk** [comment]
in my experiment，my experiment accuracy is only 81%. i use 5e-3 learning rate,5e-4 weight decay , 200 and 300 epoch decrease learning rate with factor 10,stop at 350 epoch, use tvl1 flow, and pretrained with inception v2 in tensorflow , and my training accuracy is about 97% without parital batch norm.The result is wierd

**Comment by yjxiong** [comment]
Not sure how you implemented it in TensorFlow. This codebase is for PyTorch.

**Comment by zane-wong-developer** [comment]
Hi, @imnotk , I now reproduce TSN use Tensorflow, But I have no idea about how to copy the mean of Parameters to first convolution layer, could you give me some suggestion ?

**Comment by imnotk** [comment]
@Flyzanetothesky just repeat its mean value of channel
